app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/feedback/SessionsFeedbackFragment.kt

```diff
@@ -77,10 +82,24 @@ class SessionsFeedbackFragment : Fragment(), Injectable {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        setUpFeedbackView()
+        setUpAlertView()
+
+        sessionsFeedbackViewModel.isLoading.observe(this, {
+            if (it!!) binding.progress.toVisible() else binding.progress.toGone()
+        })
+    }
+
+    private fun setUpFeedbackView() {
         sessionsFeedbackViewModel.sessionFeedback.observe(this, { result ->
             when (result) {
                 is Result.Success -> {
                     binding.sessionFeedback = result.data
+                    binding.submit.text =
+                            if (binding.sessionFeedback!!.submitted)
```

Comment:
I think we can assign local variable for result.data
What do you think?

```
val feedback = result.data
binding.sessionFeedback = feedback
if (feedback.submitted)
```

Reply:
Please use braces 👀 
https://android.github.io/kotlin-guides/style.html#expressions